output/cloud/expv2/flush.go

```diff
@@ -1,16 +1,113 @@
 package expv2
 
-import "context"
+import (
+	"context"
 
-type noopFlusher struct {
-	referenceID string
-	bq          *bucketQ
+	"go.k6.io/k6/metrics"
+	"go.k6.io/k6/output/cloud/expv2/pbcloud"
+)
 
-	// client      MetricsClient
+type metricsCloudClient interface {
+	Push(ctx context.Context, referenceID string, samples *pbcloud.MetricSet) error
 }
 
-func (f *noopFlusher) Flush(ctx context.Context) error {
+type metricsFlushing struct {
```

Comment:
```suggestion
type metricsFlusher struct {
```

In this case, and judging by how it's used, I suggest keeping the same naming form as before and ending the name with `-er`.

Reply:
Done